lib/__tests__/standalone-globs.test.js

```diff
@@ -1,13 +1,5 @@
 'use strict';
 
-/* eslint-disable node/no-extraneous-require */
-
-const describe = require('@jest/globals').describe;
-const expect = require('@jest/globals').expect;
-const it = require('@jest/globals').it;
-
-/* eslint-enable */
-
```

Comment:
[note] This change https://github.com/jest-community/eslint-plugin-jest/pull/1094 may affect, but I don't know the details.

I think removing `@jest/globals` has no problems since it's used only in the test file.